blackbox/docs/release_notes/1.2.0.txt

```diff
@@ -188,4 +193,128 @@ Fixes
 
       SELECT count(*) from t1, t2 order by 1
 
+
+Upgrade Guide
+=============
+
+Daemon User
+-----------
+
+You can no longer run CrateDB as the superuser on Unix-like systems. You should
+create a new ``crate`` user for running the CrateDB daemon.
+
+Logging
+-------
+
+The ``logging.yml`` has been removed. You must migrate your :ref:`conf-logging`
+configuration to the new ``log4j2.properties`` file.
+
+System Properties
+-----------------
+
+You can no longer use the ``JAVA_OPTIONS`` or ``CRATE_JAVA_OPTS`` environment
+variables to pass configuration to CrateDB itself, for example::
+
+    JAVA_OPTIONS=-Dcluster.name=crate
+
+Or::
+
+    CRATE_JAVA_OPTS=-Dcluster.name=crate
+
+Instead, you must pass these options in on the :ref:`cli`.
+
+You can continue to use the ``JAVA_OPTIONS`` and ``CRATE_JAVA_OPTS``
+environment variables to set general JVM properties and CrateDB specific JVM
+properties, respectively.
+
+Configuration Changes
+---------------------
+
+Many configuration settings and files have been renamed or removed. You must
+review the `Breaking Changes`_ section above and update your setup as
+necessary.
+
+SQL Changes
+-----------
+
+Several breaking changes were made to CrateDB's SQL. This includes changes to
+time parsing, syntax changes, and new reserved keywords. You must review the
+`Breaking Changes`_ section above and update your client code as necessary.
+
+
+Bind Address
+------------
+
+The default bind address has been changed from `0.0.0.0` to the loopback
+address (meaning it will only be accessible on ``localhost``).
+
+If you bind to a network reachable IP address (e.g. you're configuring your
+cluster in a production or staging environment and not just for development
+purposes on your local machine) you must add the following line to your
+:ref:`configuration` file::
+
+    network.host: 0.0.0.0
+
+.. NOTE::
+
+   If you bind to a network reachable IP address, you must follow the
+   instructions in the next section.
+
+Elasticsearch Bootstrap Checks
```

Comment:
@joemoe mentioned this too

I guess it depends how the bootstrap checks appear to the end user and whether we plan to document in the main documentation as something to prepare for, and so people can learn about them without looking at this particular set release notes (I think we should)

in fact, the above ^^ suggests that maybe this specific section of the releases notes should be created as a standalone part of the documentation, and in fact, in these release notes, we say "1.2.0 introduced bootrstrap checks. please consult Bootstrap Checks for details and so you can make the necessary changes to your setup"

@dobe thoughts?